src/main/msp/msp.c

```diff
@@ -2021,6 +2021,91 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
     return !unsupportedCommand;
 }
 
+
+#ifdef USE_SIMPLIFIED_TUNING
+// reads simplified PID tuning values from MSP buffer
+static void readSimplifiedPids(pidProfile_t* pidProfile, sbuf_t *src) {
+    pidProfile->simplified_pids_mode = sbufReadU8(src);
+    pidProfile->simplified_master_multiplier = sbufReadU8(src);
+    pidProfile->simplified_roll_pitch_ratio = sbufReadU8(src);
+    pidProfile->simplified_i_gain = sbufReadU8(src);
+    pidProfile->simplified_d_gain = sbufReadU8(src);
+    pidProfile->simplified_pi_gain = sbufReadU8(src);
+#ifdef USE_D_MIN
+    pidProfile->simplified_dmin_ratio = sbufReadU8(src);
+#else
+    sbufReadU8(src);
+#endif
+    pidProfile->simplified_feedforward_gain = sbufReadU8(src);
+    pidProfile->simplified_pitch_pi_gain = sbufReadU8(src);
+    sbufReadU8(src); // reserved for future usage
```

Comment:
Would we not be better off keeping this simple for now and up revving in future should we need to add more fields? We're struggling for code space, so having stuff like this just in case doesn't feel right to me.

Reply:
@SteveCEvans  thanks for looking into PR.
Here are the memory measurements, i guess this is not me to decide, i do whatever you guys think is good.
![image](https://user-images.githubusercontent.com/2925027/145697811-819c9bb9-ff58-4cb2-ac47-c6b8adadd031.png)

Reply:
Thanks for the numbers. We are running right on the ragged edge here at over 99% FLASH utilisation. Using any of that remaining 1% without a good reason would seem profligate to me!

Reply:
ok, removed :)